src/pages/billing/Project.js

```diff
@@ -423,17 +454,35 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
   useEffect(() => { StateHistory.update({ projectUsers }) }, [projectUsers])
   // Update cost data only if report date range changes, or if spend report tab was selected.
   useEffect(() => {
-    const maybeLoadTotalCost = _.flow(
+    const maybeLoadProjectCost = _.flow(
       reportErrorAndRethrow('Unable to retrieve spend report data'),
       Utils.withBusyState(setUpdating)
     )(async () => {
-      if (!updatingTotalCost && totalCost === null && tab === spendReportKey) {
-        setUpdatingTotalCost(true)
+      if (!updatingProjectCost && projectCost === null && tab === spendReportKey) {
+        setUpdatingProjectCost(true)
         const endDate = new Date().toISOString().slice(0, 10)
-        const startDate = new Date((Date.now() - spendReportLengthInDays * 24 * 60 * 60 * 1000)).toISOString().slice(0, 10)
+        const startDate = addDays(-1 * spendReportLengthInDays, new Date()).toISOString().slice(0, 10)
```

Comment:
@aherbst-broad I switched to using this library. Of course, that means we have to remember to update the library if DST goes away!

Reply:
`subDays`?

Reply:
When I saw `addDays`, and read that it could take a negative number, I figured they surely didn't have a method specifically for subtracting days. :) But I changed itl.